eclipse-collections-api/src/main/java/org/eclipse/collections/api/map/ConcurrentMutableMap.java

```diff
@@ -43,4 +44,10 @@ default ConcurrentMutableMap<K, V> withMapIterable(MapIterable<? extends K, ? ex
         this.putAllMapIterable(mapIterable);
         return this;
     }
+
+    @Override
+    default void forEach(BiConsumer<? super K, ? super V> action)
+    {
+        this.forEachKeyValue(action::accept);
+    }
```

Comment:
Does this really need an override? Wouldn't the version inherited from MapIterable or MutableMapIterable be fine?

If this is an inheritance problem, then it should refer to a superclass.

```java
    @Override
    default void forEach(BiConsumer<? super K, ? super V> action)
    {
        MutableMap.super.forEach(action);
    }
```

Reply:
Yup, it is an inheritance issue. Thanks!